redmine_import_time_entry/unit/mapper.py

```diff
@@ -64,11 +66,20 @@ def account_id(self, record):
                 })
 
         project = projects[0]
-
-        return {
+        res = {
             'project_id': project.id,
         }
 
+        if record.get('version'):
+            task = self.env['project.task'].search([
+                ('project_id', '=', project.id),
+                ('name', '=', record['version'])
+            ])
+            if len(task) == 1:
+                res['task_id'] = task.id
```

Comment:
The title says _Redmine Versions will be synchronized_ but they are only bound by the `task.id`
An additional import is needed.

Thank you for finding the time to reach this point.